### PR TITLE
awsome_bookES6_feature

### PR DESCRIPTION
**Pull request by HeDevedUp**
this project was bootstrapped from the previous version of the awesome book code
For this project, I used the es6 module and restructured my code to the modular architecture  
where I had a module folder where all my code is broken into component .

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import bookData from './modules/Books';
+import Books from './modules/Books.js';
```
Comment (Ismailco) [inline index.js]: https://github.com/HeDevedUp/wireframeEs6/blob/461576febc4f559ca5256274793f4ee39b8f336a/modules/list_books.js#L3

- [ ] You should use the ``` arrow``` function for this project, it's a requirement to use ES6 syntax, please change all the ```functions``` to ``` arrow functions```

Comment (Ismailco) [inline index.js]: - [ ] The project can't be run, please consider implementing the run script into ```packeage.json```

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @HeDevedUp,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @HeDevedUp👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- ES6 syntax ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**